lib/membrane/live_compositor/output_processor.ex

```diff
@@ -31,6 +46,28 @@ defmodule Membrane.LiveCompositor.VideoOutputProcessor do
   def handle_stream_format(_pad, _stream_format, _ctx, state) do
     {[stream_format: {:output, state.output_stream_format}], state}
   end
+
+  @impl true
+  def handle_event(:input, event, _ctx, state) do
+    {[event: {:output, event}], state}
+  end
+
+  @impl true
+  def handle_event(:output, %Membrane.KeyframeRequestEvent{}, _ctx, state) do
+    case ApiClient.request_keyframe(state.lc_port, state.output_id) do
```

Comment:
I'm not an expert, but I understand the "proper" way of handling sth like this in elixir would be to send a message to the parent process, and that process should send the actual request.